### PR TITLE
add support for offset in `#[asset(texture_atlas(...))]`

### PR DESCRIPTION
**Pull request by FraserLee**
Seems like access to this parameter was missing, with a default `Some(Vec2::splat(0.))` given instead of any way to access the value. Hope this helps!

### CODE REVIEW

File: bevy_asset_loader_derive/src/assets.rs [hunk 1]
```diff
@@ -609,7 +619,9 @@ mod test {
                 columns: 10,
                 rows: 5,
                 padding_x: 2.0,
-                padding_y: 0.0
+                padding_y: 0.0,
+                offset_x: 0.0,
```
Comment (NiklasEi) [inline bevy_asset_loader_derive/src/assets.rs:623]: Could you add a value for one of the offset fields like I did for padding?

**Review by NiklasEi — COMMENTED** [review]
Two nitpicks, otherwise good to be merged. Thank you :+1:

**Comment by FraserLee** [comment]
thanks! Lmk if you need anything else.

**Review by NiklasEi — APPROVED** [review]
I am getting the same CI failures on the `main` branch, so I'll take care of that later.

**Pull request merged**